packages/ra-core/src/controller/edit/useEditController.ts

```diff
@@ -116,9 +118,13 @@ export const useEditController = <RecordType extends RaRecord = any>(
         ) =>
             Promise.resolve(
                 transformFromSave
-                    ? transformFromSave(data)
+                    ? transformFromSave(data, {
```

Comment:
check the linter warning for missing dependency

Reply:
Thank you I've not noticed. It's green now.